x-pack/plugins/reporting/server/usage/decorate_range_stats.ts

```diff
@@ -9,19 +9,22 @@ import { uniq } from 'lodash';
 import {
   CSV_JOB_TYPE,
   CSV_JOB_TYPE_DEPRECATED,
+  DEPRECATED_JOB_TYPES,
   PDF_JOB_TYPE,
   PNG_JOB_TYPE,
 } from '../../common/constants';
 import { AvailableTotal, ExportType, FeatureAvailabilityMap, RangeStats } from './types';
 
+const jobTypeIsDeprecated = (jobType: string) => DEPRECATED_JOB_TYPES.includes(jobType);
+
 function getForFeature(
   range: Partial<RangeStats>,
   typeKey: ExportType,
   featureAvailability: FeatureAvailabilityMap,
   additional?: any
```

Comment:
I took a crack at fixing this `any` but was not able to see a clean fix for it yet

Reply:
You can use generics as a clean fix.

Reply:
@dokmic I merged your suggestion and there is still a lot of ambiguity in the body of this function. I reverted the commit for now, but let's keep track of this. It is a very nasty `any` and in a big way, it indicates the logic of the function could be improved.